R/pal-brewer.r

```diff
@@ -25,9 +28,14 @@ brewer_pal <- function(type = "seq", palette = 1) {
   # number of items.
   function(n) {
     if (n < 3)
-      suppressWarnings(RColorBrewer::brewer.pal(n, pal))[seq_len(n)]
+      pal <- suppressWarnings(RColorBrewer::brewer.pal(n, pal))[seq_len(n)]
     else
-      RColorBrewer::brewer.pal(n, pal)[seq_len(n)]
+      pal <- RColorBrewer::brewer.pal(n, pal)[seq_len(n)]
+
+    if (direction == -1)
+      pal <- rev(pal)
+
+    return(pal)
```

Comment:
OK. I thought it was good practice to always have an explicit `return()`(it's certainly less confusing for newcomers/students in my experience).